.github/workflows/tests.yml

```diff
@@ -10,10 +10,12 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [8.1, '8.0']
-                laravel: [9.*, 8.*]
+                php: [8.2, 8.1]
```

Comment:
@manavo bring 8.0 back here as well :)

Reply:
Not sure how to do that, without the know failure case of Laravel 10 and PHP 8?

Reply:
you can add an exclude parameter on the matrix to exclude combinations that fail

```
exclude:
      - php: 8.0
        laravel: 10.*
```

Reply:
Got it :)